core/integrations/sentry_client.py

```
@@ -0,0 +1,204 @@
+import requests
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)